CODE_OF_CONDUCT.md

```diff
@@ -4,8 +4,10 @@ This Code of Conduct is adapted from [Rust's wonderful
 CoC](http://www.rust-lang.org/conduct.html).
```

Comment:
I believe Rust adapted this from the [Contributor Covenant] so that might be worth including in the first sentence about adaptations?

[Contributor Covenant]: contributor-covenant.org